symphony-bdk-core/src/test/java/com/symphony/bdk/core/auth/impl/OboAuthenticatorCertImplTest.java

```diff
@@ -0,0 +1,101 @@
+package com.symphony.bdk.core.auth.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.symphony.bdk.core.api.invoker.ApiRuntimeException;
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.test.MockApiClient;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class OboAuthenticatorCertImplTest {
+
+  private static final String APP_AUTHENTICATE = "/sessionauth/v1/app/authenticate";
+  private static final String APP_AUTHENTICATE_OBO_USERNAME = "/sessionauth/v1/app/username/{username}/authenticate";
+  private static final String APP_AUTHENTICATE_OBO_USERID = "/sessionauth/v1/app/user/{uid}/authenticate";
+
+  private OboAuthenticatorCertImpl authenticator;
+  private MockApiClient mockApiClient;
+
+  @BeforeEach
+  void init() {
+    mockApiClient = new MockApiClient();
```

Comment:
If you use `this` for the `authenticator` then your should use `this` for the `mockApiClient` as well